src/resolver/message.ts

```diff
@@ -1,26 +1,26 @@
 export class Message {
   public static readonly AUDIOBOOK_TAG = 'audiobook';
 
-  private tags: string[];
-  private title: string | null;
+  public static tags: string[]; //made it public to display request type in the inline preview
```

Comment:
You should not use static objects! 
This way two contemporary requests share them.